test/integration/flags.go

```diff
@@ -32,7 +32,7 @@ func TestMain(m *testing.M) {
 	os.Exit(m.Run())
 }
 
-var startTimeout = flag.Int("timeout", 25, "number of minutes to wait for minikube start")
+var startTimeout = flag.Duration("timeout", 25*time.Minute, "max duration to wait for a full minikube start")
```

Comment:
8x the default we give to users is crazy. I can understand 2x, but more than that I feel like we are making ourselves reliable at the expense of users.

Reply:
sorry for the confusion, the --wait-time out is Per component not for all start.

before this PR the total wait was 5 x Per component
this PR reduces it to 3 min for end user (per component)

so it is not too much more than what we do for the end user. (if we count 5* per component")

but I intend to have another clean up PR in the integration tests ,to get rid of all kind of Retrying Start in parallel. once fixed all the flakes ( certs, and corruptions...)